components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -462,6 +465,12 @@ be lost if SCREAM_HACK_XML is not enabled.
     <nc hgrid="ne256np4|ne1024np4">0.0</nc>
     <ni hgrid="ne256np4|ne1024np4">0.0</ni>
     <o3_volume_mix_ratio hgrid="ne0np4_conus_x4v1_lowcon">0.0</o3_volume_mix_ratio>
+
+    <!-- Information about perturbed fields-->
+    <perturbed_fields type="array(string)"/>
```

Comment:
Can you add a `doc` attrib for the new parameters?

Note: if the same param appears 2+ times (like `perturbed_fields`), put the doc string only in the first occurrence.